mlflow/johnsnowlabs/__init__.py

```diff
@@ -665,7 +665,7 @@ def _load_model(model_uri, dfs_tmpdir_base=None, local_model_path=None):
     return nlp.load(path=local_model_path)
 
 
-def load_model(model_uri, dfs_tmpdir=None, dst_path=None, **kwargs):  # noqa: D417
+def load_model(model_uri, dfs_tmpdir=None, dst_path=None, **kwargs):
```

Comment:
```suggestion
def load_model(model_uri, dfs_tmpdir=None, dst_path=None):
```

let's remove this. It's unused.